### PR TITLE
added code coverage workflow, fixes #6

### LINKED ISSUE #6: Add code coverage
Basically, do this: https://about.codecov.io/blog/javascript-code-coverage-using-github-actions-and-codecov/

### PR DESCRIPTION
**Pull request by sapatgit**
This PR fixes #6 
The action was working and generated the following [report](https://app.codecov.io/gh/sapatgit/angular-material-starter-template/branch/add-code-coverage-workflow/commits?page=1) 
Let me know if any changes needed

### CODE REVIEW

**Comment by sardapv** [comment]
@sapatgit Thanks for the PR :), looks cool. @sjetha can you please review.

**Comment by sardapv** [comment]
@sapatgit can we have custom command to run on local as well? Would be great with that option. I think you can add it in package.json

**Comment by sapatgit** [comment]
@sardapv sure I can work on that

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```
Comment (jethas-bennettjones) [inline .github/workflows/coverage.yml:11]: Honestly, not a fan of this matrix strategy if you're not actually doing anything matrix-y.

**Comment by jethas-bennettjones** [comment]
One minor issue with the yank file, but it looks good otherwise.

**Review by jethas-bennettjones — CHANGES_REQUESTED** [review]

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (fasterinnerlooper) [inline .github/workflows/coverage.yml]: Honestly, not a fan of the Matrix strategy if you’re not doing anything matrix-y.

**Review by fasterinnerlooper — CHANGES_REQUESTED** [review]

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (sapatgit) [reply .github/workflows/coverage.yml]: @sjetha Hi, can you check now? I have modified the PR

**Comment by sapatgit** [comment]
@sardapv Hey, I have added the local coverage check script. Any comments or suggestions?

**Review by sardapv — APPROVED** [review]

**Comment by sardapv** [comment]
@all-contributors Please add @sapatgit  for docs, code

**Pull request merged**